### ISSUE TITLE
Can i use Horovod for inference on clusters?

### ISSUE THREAD

**Issue by liuyongheng123** [horovod/horovod#1242]
**Is your feature request related to a problem? Please describe.**  No

**Describe the solution you'd like**——yes
Whether Horovod  can  be used for inference in the Raspberry pi Cluster or other AI accelerate cluster？

My cluster is based on a bunch of Raspberry PI, which means they are very poor at computing. So I am only using them for inference for a benchmark. 

As I know, Horovod allocates data /resource and then collect the gradients for gradient descent in the training process. But I want to use Horovod for inference, which means I do NOT need the gradients calculated. I only need to know the predictions based on the loaded model. Is there any available API in Horovod now to do that ?    

**Describe alternatives you've considered**

**Additional context**

**Comment by abditag2** [comment]
Not really. Horovod only helps with distributed training stage. Once the model is trained, the user can serve it just like a standard TF/PyTorch model.

**Comment by SAswinGiridhar** [comment]
@liuyongheng123 May I know how you were able to solve the multi gpu inference scenario?

**Comment by liuyongheng123** [comment]
Hello, happy to hear your asking!
In my experiment, I get three Inference development board with AI inference chip , which the Ubuntu runs on. I install Horovod and openmpi on the OS of the board , and chose one of the three as master nodes run the "horovodrun" command. The three board differentiated to classify the part load of picture which was  decomposed to three part.
After the experiment, I understood abditag2's words.He is right. 
In inference scenario, what Horovod do is also done by shell script easily!
Thanks to abditag2 again

**Comment by abditag2** [comment]
@SAswinGiridhar Horovod does not provide any tools for serving. You can take a look at https://github.com/uber/neuropods which is another Uber project that helps package the DL models for serving.
